lib/main.coffee

```diff
@@ -395,30 +397,44 @@ module.exports = Hydrogen =
     inspect: ->
         language = @editor.getGrammar().name.toLowerCase()
 
-        if @editor.getSelectedText() != ''
-            code = @editor.getSelectedText()
-            cursor_pos = code.length
-        else
-            cursor = @editor.getLastCursor()
-            row = cursor.getBufferRow()
-            code = @getRow(row)
-            cursor_pos = cursor.getBufferColumn()
+        [code, cursor_pos] = @getCodeToInspect()
 
         KernelManager.inspect language, code, cursor_pos, (result) ->
             console.log 'inspect result:', result
             found = result['found']
             if found is true
                 data = result['data']
-                atom.workspace.open('.Hydrogen Inspector', {split:'down'}).then (editor) ->
-                    if editor.isEmpty() is false
-                        buffer = editor.getBuffer()
-                        buffer.deleteRows(0, buffer.getLineCount())
-                    editor.setSoftWrapped(true)
-                    editor.setLineNumberGutterVisible(false)
-                    editor.insertText(stripAnsi(data['text/plain']))
-                    editor.moveToTop()
-                    atom.workspace.activatePreviousPane()
-                    # Hack for easy closing
-                    editor.save()
+                lines = stripAnsi(data['text/plain']).split('\n')
```

Comment:
On a side note, transformime currently has to get all the transforms bundled for a full Jupyter setup: https://github.com/jupyter/jupyter-js-notebook/blob/476fbf8233d2fa87ee0495f769a949eb2a271b67/src/output-area/widget.ts#L121-L136